<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModbusTestClass</class>
 <widget class="QMainWindow" name="ModbusTestClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ModbusTest</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>601</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="3">
      <widget class="QLabel" name="m_lData">
       <property name="text">
        <string>数据:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="m_lRegisterAddress">
       <property name="text">
        <string>起始地址：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="4">
      <widget class="QLineEdit" name="m_DlineEdit"/>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="m_RAlineEdit"/>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="6">
      <widget class="QPushButton" name="m_btnWrite">
       <property name="text">
        <string>写入</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>601</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QTextEdit" name="m_RDtextEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="1" column="2">
      <widget class="QLabel" name="m_SerialPort">
       <property name="text">
        <string>串口号：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="m_SPlineEdit"/>
     </item>
     <item row="1" column="4">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="1" column="5">
      <widget class="QLabel" name="m_DeviceAddress">
       <property name="text">
        <string>设备地址：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="m_BRlineEdit"/>
     </item>
     <item row="1" column="4">
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="m_BaudRate">
       <property name="text">
        <string>波特率：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QLineEdit" name="m_DAdrlineEdit"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ModbusTest.qrc"/>
 </resources>
 <connections/>
</ui>
